root/static/scripts/edit/externalLinks.js

```diff
@@ -545,7 +545,17 @@ export class ExternalLinksEditor
       };
     } else if (isNewOrChangedLink && !isValidURL(link.url)) {
       error = {
-        message: l('Enter a valid url e.g. "http://google.com/"'),
+        message: exp.l('Please enter a valid URL, such as “{example_url}”.',
+                       {example_url: <span className="url-quote">{'http://example.com/'}</span>}),
+        target: URLCleanup.ERROR_TARGETS.URL,
+      };
+    } else if (isNewOrChangedLink && isExample(link.url)) {
+      error = {
+        message: exp.l(
+          `“{example_url}” is just an example.
+          Please enter the actual link you want to add.`,
+          {example_url: <span className="url-quote">{link.url}</span>},
+        ),
```

Comment:
Do we think people are going to try to enter `http://example.com/` just because the previous message references it?

I think they'd have to be incredibly confused about what they're doing to attempt that -- so much so that I dunno if having a dedicated message for this will suddenly enlighten them. :D

Reply:
Well, given a bunch of people seemed to enter the Google link from the previous example, it can't hurt to play it safe...

Reply:
I agree it might be seen as insulting the intelligence of editors, but mistakes happen and it seems worth explaining why the URL is being rejected.